packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -611,6 +611,9 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
             this.channelDeferred.resolve(this.channel);
         } catch (error) {
             this.channelDeferred?.reject(error);
```

Comment:
Side note - Can you explain why this codepath doesn't use `CreateProcessingError`?

Reply:
DataProcessingError does not have comment describing when it should be used, but I'd guess the intention here is to track op processing errors, and that's not op processing error (most errors here are likely impacting local client only and are user errors).
That's my guess.

Reply:
Ok looking closer at this code, I guess this function is mostly runtime mechanics, plus pending op playback (any any errors in there will already be classified as DataProcessingError).  Not sure I see where user errors would arise, seems like mostly our own code. 🤷